### ISSUE TITLE
unclear technical note on AlphaFold v2.3.0

### ISSUE THREAD

**Issue by RJ3** [deepmind/alphafold#750]
In the [Technical Note](https://github.com/deepmind/alphafold/blob/main/docs/technical_note_v2.3.0.md) it states the following:

> In cases where the stoichiometry is unknown, such as in genome-scale prediction, it is likely that single chain AlphaFold will be more accurate on average unless the chain has several thousand residues.

Does this mean that if there's a chance your single chain is part of a multimer with unknown stoichiometry then the older version of AlphaFold will be more accurate?

If you're just looking at monomers, then the new AlphaFold v.2.3.0 in multimer mode to be preferred?

**Comment by patrickbryant1** [comment]
Yes - but with modifications. The issue is how to get the stoichiometry for anything with >2 chains though. If you have this, chances are you have the structure already.
